src/pages/TeacherDashboard/TeacherDashboard.jsx

```diff
@@ -1,21 +1,21 @@
-import React, {useState, useEffect} from 'react';
+import {useState, useEffect} from 'react';
 import styles from "./TeacherDashboardStyles.module.css"
 import SideBar from "../../components/SideBar/SideBar";
 import {useNavigate} from "react-router-dom";
 
 const TeacherDashboard = () => {
-    const apiUrl = import.meta.env.VITE_API_BASE_URL;
+    const baseUrl = import.meta.env.VITE_BASE_URL;
     const [courses, setCourses] = useState([]);
     const token = localStorage.getItem('token');
     const navigate = useNavigate();
 
     useEffect(() => {
         if (!token) {
-            navigate('/');
+            navigate('/frontpage');
```

Comment:
When you rebase the new changes, this will need to be '/' again since I moved the FrontPage component to that route.